builds/misc/images.yaml

```diff
@@ -375,5 +429,9 @@ jobs:
       displayName: 'Publish Edge Hub Manifest'
     - script: scripts/linux/buildManifest.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(Build.BuildNumber) -t $(System.DefaultWorkingDirectory)/edge-modules/SimulatedTemperatureSensor/docker/manifest.yaml.template -n microsoft --tags "$(tags)"
       displayName: 'Publish Temperature Sensor Manifest'
+    - script: scripts/linux/buildManifest.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(Build.BuildNumber) -t $(System.DefaultWorkingDirectory)/edgelet/iotedged/docker/manifest.yaml.template -n microsoft --tags "$(tags)"
```

Comment:
Nit: why need "/iotedged/" in "-t $(System.DefaultWorkingDirectory)/edgelet/iotedged/docker/manifest.yaml.template"?

Reply:
iotedged docker files are different for example from iotedge-diagnostics. They are using different base images one use glibc based and other use musl based.